src/rgw/rgw_admin.cc

```diff
@@ -2889,11 +2873,11 @@ static int bucket_sync_status(rgw::sal::Driver* driver, const RGWBucketInfo& inf
 
   for (auto& zone_id : zone_ids) {
     auto z = static_cast<rgw::sal::RadosStore*>(driver)->svc()->zone->get_zonegroup().zones.find(zone_id.id);
-    if (z == static_cast<rgw::sal::RadosStore*>(driver)->svc()->zone->get_zonegroup().zones.end()) { /* should't happen */
+    if (z == static_cast<rgw::sal::RadosStore*>(driver)->svc()->zone->get_zonegroup().zones.end()) { /* shouldn't happen */
```

Comment:
there was a request to expand the `mustn't` contraction. I can expand others, but given how large this change set is, I'd rather do that separately. (I can if requested, it isn't a big deal to me, it's just more for people to review sooner.)